plugins/module_utils/foreman_helper.py

```diff
@@ -1072,14 +1033,14 @@ def _foreman_spec_helper(spec):
             if not flat_name:
                 flat_name = '{0}_id'.format(key)
             foreman_value['resource_type'] = value.get('resource_type', HAS_APYPIE and inflector.pluralize(key))
-            foreman_value.update({k: v for (k, v) in value.items() if k in ['thin', 'scope', 'search_by', 'search_operator', 'resolve']})
+            foreman_value.update({k: v for (k, v) in value.items() if k in ['failsafe', 'resolve', 'scope', 'search_by', 'search_operator', 'thin']})
```

Comment:
`['failsafe', 'resolve', 'scope', 'search_by', 'search_operator', 'thin']` should be in a `_STATIC` ;)